meta-balena-common/recipes-core/healthdog/healthdog_1.0.1.bb

```diff
@@ -0,0 +1,58 @@
+# Auto-Generated by cargo-bitbake 0.3.12
+#
+inherit cargo
+
+# If this is git based prefer versioned ones if they exist
+# DEFAULT_PREFERENCE = "-1"
+
+# how to get healthdog could be as easy as but default to a git checkout:
+# SRC_URI += "crate://crates.io/healthdog/1.0.1"
+SRC_URI += "git://git@github.com/balena-os/healthdog-rs.git;protocol=ssh;nobranch=1"
```

Comment:
I find a http remote works better for public repos, as it means you don't need ssh keys on any build servers.